conformance/lang/expressions/mapping_constructor_expr.balt

```diff
@@ -0,0 +1,2080 @@
+Test-Case: parser-error
+Description: Test syntax errors in mapping constructor.
+Labels: any, mapping-constructor-expr, map-type, string
+
+function errorFunction() {
+    map<any> _ = {5 : 5}; // @error field name should be an identifier
```

Comment:
Can you recheck errors here. They are so different from actual values. As an example first error is `incompatible types: expected 'string', found 'int'`